docs/en/changes/changes.md

```diff
@@ -28,6 +28,8 @@
 * Fix the problem that some configurations (such as group.id) did not take effect due to the override order when using the kafkaConsumerConfig property to extend the configuration in Kafka Fetcher.
 * Remove build time from the OAP version.
 * Add data-generator module to run OAP in testing mode, generating mock data for testing.
+* Support receive Kubernetes processes from gRPC protocol.
+* Fix create eBPF Profiling task verify failure.
```

Comment:
I think this feature doesn't exist in 9.0, then we don't need this in the change log.

Reply:
Deleted.